README.md

```diff
@@ -209,7 +209,9 @@ Hokusai is currently tested on Pythons 2.7.16 and 3.5.8.
 
 2) Install dependencies: `poetry install --no-root`.
 
-3) Run tests
+3) make sure you installed minikube with `brew install minikube` and run with `minikube start `
```

Comment:
Nitpick: could you explain why `minikube` is necessary? I don't have `minikube` set up and never noticed any issues (but I also haven't developed within hokusai!)

Reply:
Yes it fixed the broken setup for me for some reason. I honestly don't understand why, but it was the only thing that made it work.